src/scripts/main.js

```diff
@@ -1,3 +1,30 @@
 'use strict';
 
 // write your code here
+const h1 = document.querySelector('.logo');
+const body = document.body;
+
+const promSuccess = () => {
+  return new Promise((resolve) => {
+    const div = document.createElement('div');
+
+    div.classList.add('message');
+    div.textContent = 'Promise was resolved';
+    body.append(div);
+    resolve();
+  });
+};
```

Comment:
The function `promSuccess` should be used to resolve `promise1`. You should create `promise1` and attach this function as its success handler.